assets/app/view/game/spreadsheet.rb

```diff
@@ -270,6 +270,14 @@ def render_corporation(corporation, operating_order, current_round, index)
         ])
       end
 
+      def par_price?(corporation)
```

Comment:
should you just put the closed in the corporation getter intsead?

Reply:
Ok, have now put some logic in the par_price and share_price getters.
Had to change accessor method for share_price